frontend/src/app/contacts/contacts-list/contacts-list.component.html

```diff
@@ -68,5 +68,8 @@
     <div class="blocks"><button button (click)="openAddDialog()" mat-raised-button color="primary">
             <mat-icon>add</mat-icon> Kontakt hinzufügen
         </button>
+        <button class="button" *ngIf="isAdminUserLoggedIn" mat-raised-button color="primary" title="Dummy Kontakt hinzufügen" (click)="addDummyContact()"> 
+            <mat-icon>add</mat-icon> Dummy Kontakt hinzufügen
```

Comment:
wollen wir wirklich so einen Button in der "Live" Webseite?

Reply:
jap. weil der ist auch nur als admin sichtbar